google-cloud-spanner/src/main/java/com/google/cloud/spanner/BatchClientImpl.java

```diff
@@ -72,7 +79,9 @@ public BatchReadOnlyTransaction batchReadOnlyTransaction(TimestampBound bound) {
   @Override
   public BatchReadOnlyTransaction batchReadOnlyTransaction(BatchTransactionId batchTransactionId) {
     SessionImpl session =
-        sessionClient.sessionWithId(checkNotNull(batchTransactionId).getSessionId());
+        sessionClient.sessionWithId(
+            checkNotNull(batchTransactionId).getSessionId(),
+            batchTransactionId.isMultiplexedSession());
```

Comment:
We should not pass in multiplexed true/false here, because this client does not know anything about this session. The session might have been created by a different client and shared out-of-band with this client. The fact that this client uses multiplexed sessions does not mean that the other client uses multiplexed sessions (and vice-versa).